### PR TITLE
Rarity fixes, minor functional changes

### PR DESCRIPTION
**Pull request by triggrhappy0**

### CODE REVIEW

File: ptcg_cockatrice.xml [hunk 1]
```diff
@@ -471,7 +471,7 @@
 		</card>
 		<card>
 			<name>Dugtrio</name>
-			<set picURL="https://images.pokemontcg.io/base4/71_hires.png" rarity="common">PTCG</set>
+			<set picURL="https://images.pokemontcg.io/base4/71_hires.png" rarity="ucommon">PTCG</set>
```
Comment (brooks42) [inline ptcg_cockatrice.xml:474]: many of these are `ucommon`

File: ptcg_cockatrice.xml [hunk 2]
```diff
@@ -795,7 +795,7 @@
 		</card>
 		<card>
 			<name>Dodrio</name>
-			<set picURL="https://images.pokemontcg.io/ecard2/73_hires.png" rarity="common">PTCG</set>
+			<set picURL="https://images.pokemontcg.io/ecard2/73_hires.png" rarity="ucommon">PTCG</set>
```
Comment (brooks42) [inline ptcg_cockatrice.xml:798]:
```suggestion
			<set picURL="https://images.pokemontcg.io/ecard2/73_hires.png" rarity="uncommon">PTCG</set>
```

File: ptcg_cockatrice.xml [hunk 3]
```diff
@@ -673,15 +673,15 @@
 		</card>
 		<card>
 			<name>Tentacruel</name>
-			<set picURL="https://images.pokemontcg.io/base6/96_hires.png" rarity="common">PTCG</set>
+			<set picURL="https://images.pokemontcg.io/base6/96_hires.png" rarity="ucommon">PTCG</set>
```
Comment (brooks42) [inline ptcg_cockatrice.xml:676]:
```suggestion
			<set picURL="https://images.pokemontcg.io/base6/96_hires.png" rarity="uncommon">PTCG</set>
```

File: ptcg_cockatrice.xml [hunk 4]
```diff
@@ -537,7 +537,7 @@
 		</card>
 		<card>
 			<name>Primeape</name>
-			<set picURL="https://images.pokemontcg.io/base5/61_hires.png" rarity="common">PTCG</set>
+			<set picURL="https://images.pokemontcg.io/base5/61_hires.png" rarity="ucommon">PTCG</set>
```
Comment (brooks42) [inline ptcg_cockatrice.xml:540]:
```suggestion
			<set picURL="https://images.pokemontcg.io/base5/61_hires.png" rarity="uncommon">PTCG</set>
```

File: ptcg_cockatrice.xml (hunk 1 above)
Comment (brooks42) [inline ptcg_cockatrice.xml:474]:
```suggestion
			<set picURL="https://images.pokemontcg.io/base4/71_hires.png" rarity="uncommon">PTCG</set>
```

File: ptcg_cockatrice.xml [hunk 5]
```diff
@@ -988,8 +988,8 @@
 			<pt>4/5</pt>
 			<tablerow>2</tablerow>
 			<text>{1}{G}{G}{B}: Evolve from Exeggcute
-				Reach
-				Whenever Exeggutor blocks a Pokémon with flying, it deals 3 damage to that Pokémon and you draw a card.</text>
+				Whenever Exeggutor blocks a Pokémon with flying, it deals 3 damage to that Pokémon and you draw a card.
```
Comment (brooks42) [inline ptcg_cockatrice.xml:991]: I thought keywords always went first

File: ptcg_cockatrice.xml (hunk 1 above)
Comment (triggrhappy0) [inline ptcg_cockatrice.xml]:
```suggestion
			<set picURL="https://images.pokemontcg.io/base4/71_hires.png" rarity="uncommon">PTCG</set>
```

Comment (triggrhappy0) [reply ptcg_cockatrice.xml]: Fixed

**Pull request merged**